geoapps/io/write_default_uijson.py

```diff
@@ -84,12 +84,8 @@ def write_default_uijson(path, use_initializers=False):
     }
 
     for filename, params in filedict.items():
-
-        if use_initializers:
-            if getattr(params, "forward_only", None) is not None:
-                params.write_input_file(name=filename, path=path)
-        else:
-            params.write_input_file(name=filename, path=path, default=True)
+        default = False if use_initializers else True
```

Comment:
Not to be picky, but that won't be mypy compliant. Convert to `not use_initializer` for simpler logical

Reply:
Ternary's aren't mypy compliant?

Reply:
either pylint or mypy, can't remember which but I had to replace every where on geoh5py.